app/src/main/java/com/example/drawingApp/DrawingField.kt

```diff
@@ -0,0 +1,96 @@
+package com.example.drawingApp
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import android.graphics.Color
+import android.graphics.Paint
+import android.graphics.Path
+import android.util.AttributeSet
+import android.util.Log
+import android.view.MotionEvent
+import android.view.View
+import kotlin.math.abs
+
+
+@SuppressLint("ClickableViewAccessibility")
+class DrawingField @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyle: Int = 0
+) : View(context,attrs,defStyle) {
+    /*
```

Comment:
Clean up comments

Reply:
Removed the comment as a KDoc is used instead